<a href="https://colab.research.google.com/github/ummehabiba-m/-betabytez-aiml-task1-ummeHabiba/blob/main/heart_disease_classifier.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [3]:
import pandas as pd

url = "https://raw.githubusercontent.com/kb22/Heart-Disease-Prediction/master/dataset.csv"
df = pd.read_csv(url)
df.head()

,age,sex,cp,trestbps,chol,fbs,restecg,thalach,exang,oldpeak,slope,ca,thal,target
0,63,1,3,145,233,1,0,150,0,2.3,0,0,1,1
1,37,1,2,130,250,0,1,187,0,3.5,0,0,2,1
2,41,0,1,130,204,0,0,172,0,1.4,2,0,2,1
3,56,1,1,120,236,0,1,178,0,0.8,2,0,2,1
4,57,0,0,120,354,0,1,163,1,0.6,2,0,2,1


Dataset: UCI Heart Disease
- age: age in years
- sex: 1 = male, 0 = female
- cp: chest pain type (0–3, 4 categories)
- trestbps: resting blood pressure
- chol: serum cholesterol (mg/dl)
- fbs: fasting blood sugar > 120 mg/dl (1 = true, 0 = false)
- restecg: resting ECG results (0–2)
- thalach: max heart rate achieved
- exang: exercise-induced angina (1 = yes, 0 = no)
- oldpeak: ST depression induced by exercise
- slope: slope of peak exercise ST segment
- ca: number of major vessels colored by fluoroscopy (0–4)
- thal: thalassemia (blood disorder) type
- target: 1 = has heart disease, 0 = no heart disease